.github/workflows/test.yml

```diff
@@ -51,26 +54,9 @@ jobs:
           id" -e SFDX_HUB_KEY="sample key" -e
           DB_ENCRYPTION_KEY=MMkDMBfYL0Xoz3Xu1ENs3AkdCZdJoks5PNlUBkK7KDc= web
           yarn test:py
-      - name: Upload coverage results
-        uses: actions/upload-artifact@v3
-        with:
-          name: backend-coverage
-          path: |
-            .coverage
-            coverage.xml
-
-  coverage:
-    name: Upload coverage reports
-    runs-on: ubuntu-latest
-    needs: [frontend, backend]
-    steps:
-      - uses: actions/checkout@v3
-      - name: Download frontend coverage results
-        uses: actions/download-artifact@v3
-        with:
-          name: frontend-coverage
-      - name: Download backend coverage results
-        uses: actions/download-artifact@v3
-        with:
-          name: backend-coverage
-      - uses: codecov/codecov-action@v3
+      - name: Upload coverage reports
+        run:
+          docker-compose run -e COVERALLS_REPO_TOKEN --no-deps web yarn
+          test:py:report-coverage
+      - name: Check coverage
+        run: docker-compose run --no-deps web yarn test:py:check-coverage
```

Comment:
I have a slight preference for separating the coverage reporting/checking into a separate build step than the tests themselves (so that it's clear at a glance whether the tests are actually failing vs missing coverage), but that's a minor point and there may be tradeoffs I don't understand.